### DISCUSSION TITLE
Record session on another device

### DISCUSSION THREAD

**Discussion by blufish1234** [vrcx-team/VRCX#643] [General]
If I launch VRCX on my one computer, and then play VRChat on another one, will my status be synced? (e.g. discord presence, location)

**Comment by MenacingExiler** [comment]
It should limit to what VRChat API can give since VRCX relies a lot of information on your game logs, which is only accessible when it is in same device.

**Comment by Natsumi-sama** [comment]
You can see the world you're in if you click on yourself but gameLog and Discord presence wont work.
